config/smithsonian.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Smithsonian API Base Url
+    |--------------------------------------------------------------------------
+    */
+    'api_base_url' => env('Smithsonian_API_BASE_URL', 'https://api.si.edu/openaccess/api/v1.0'),
```

Comment:
Better make "Smithsonian" in uppercase as well.

Reply:
ok. Thanks